test/test-framework/go.mod

```diff
@@ -36,4 +36,6 @@ replace (
 	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.0.0-20191016112829-06bb3c9d77c9
 )
 
+replace github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503-38ab9da00309 // Required by Helm
```

Comment:
The test-framework is a mock data of a Go project. So, why is required to add here and change its modules for the helm?

Reply:
Go project's depend on `github/operator-framework/operator-sdk`. Fro any projects resolving that dependency, this `replace` is required for resolution to succeed.

Here's what `go mod tidy` looks like in the test framework without that replace:
```console
$ go mod tidy
go: github.com/operator-framework/operator-sdk@v0.0.0 requires
	helm.sh/helm/v3@v3.0.0 requires
	github.com/deislabs/oras@v0.7.0 requires
	github.com/docker/docker@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000
```